zkg

```diff
@@ -277,11 +277,22 @@ def create_config(args, configfile):
     if not zeek_dist:
         zeek_dist = get_option(config, 'paths', 'bro_dist', '')
 
+    cmake_dir = get_option(config, 'paths', 'cmake_dir', '')
+
+    if cmake_dir:
+        # If a cmake_dir is set, inject it into any CMake invocations created
+        # from this processess via `CMAKE_PREFIX_PATH`. This allows packages to
+        # use Zeek-provided CMake modules. We append to any existing settings
+        # in the caller's environment, so existing setting keep precedence.
+        cmake_prefix_path = os.environ.get('CMAKE_PREFIX_PATH', '')
+        os.environ['CMAKE_PREFIX_PATH'] = ';'.join([cmake_prefix_path, cmake_dir])
```

Comment:
Suggesting to use `os.pathsep.join([...])` rather than `;`

Reply:
This is not a path, but a list of paths. CMake encodes lists as strings with entries separated by `;`.

Reply:
Oh. I didn't know. that. Thanks for the pointer.